framework_tvb/tvb/tests/framework/core/neocom_test.py

```diff
@@ -132,14 +132,14 @@ def test_gather_view_model_references(connectivity_index_factory, operation_fact
     storage_path = FilesHelper().get_project_folder(op.project, str(op.id))
     h5.store_view_model(sim_view_model, storage_path)
 
-    only_vm_references = h5.gather_view_model_references(sim_view_model.gid, storage_path, True)
-    all_references = h5.gather_view_model_references(sim_view_model.gid, storage_path)
+    only_vm_references = h5.gather_view_model_and_datatype_references(sim_view_model.gid, storage_path, True)[0]
```

Comment:
Please always use:
`only_vm_references, _ = h5.gather_view_model_and_datatype_references(...)`
instead of:
`only_vm_references = h5.gather_view_model_and_datatype_references(...)[0]`

Reply:
Done